core/src/main/java/org/apache/servicecomb/core/definition/MicroserviceVersionMeta.java

```diff
@@ -20,10 +20,15 @@
 import org.apache.servicecomb.core.CseContext;
 import org.apache.servicecomb.core.definition.classloader.MicroserviceClassLoaderFactory;
 import org.apache.servicecomb.serviceregistry.api.Const;
+import org.apache.servicecomb.serviceregistry.api.registry.Microservice;
 import org.apache.servicecomb.serviceregistry.consumer.MicroserviceVersion;
 
 public class MicroserviceVersionMeta extends MicroserviceVersion {
-  private MicroserviceMeta microserviceMeta;
+  MicroserviceMeta microserviceMeta;
+
+  MicroserviceVersionMeta(Microservice microservice) {
```

Comment:
microserviceMeta is not initialized, and getMicroserviceMeta may return null and cause NPE.

Reply:
`microserviceMeta` in `MicroserviceVersionMeta` is initialized in constructors and has package access modifier which ensures that the classes out of its package can only get access to it by getter method. So I think NPE can be avoid since users can only get `microserviceMeta` after it is initialized.